upgrade-1.x-to-2.x.sh

```diff
@@ -160,15 +186,19 @@ umount /var/lib/docker
 umount /resin-data
 
 # Save conf contents to /boot
-# XXX
+if [ -f "/mnt/conf/config.json" ]; then
+    # the boot partition might be mounted ro on some systems, remount
+    mount -o remount,rw "${boot_path}"
```

Comment:
Freshly provisioned `1.8.0`: 
```
|-/boot                          /dev/mmcblk0p1              vfat     ro,relatime,fmask=0022,dmask=0022,codepage=437,iocharset=ascii,shortname=mixed,errors=remount-ro
```
(and no errors in dmesg that would warrant a ro remount happening)

What do you mean by "this one is an easy catch"?